### PR TITLE
Event test

### PR DESCRIPTION
**Pull request by GGBBB2000**

### CODE REVIEW

File: MapGameController.java [hunk 1]
```diff
@@ -133,7 +136,7 @@ public void controlNPC(){
                     break;
             }
 
-            for(int Y =0;Y<15;Y++){
+            /*for(int Y =0;Y<15;Y++){
```
Comment (equal-l2) [inline MapGameController.java:139]: Don't comment-out something unneeded, but just remove it.
We can go back with git whenever we need it.

File: MapGameController.java [hunk 2]
```diff
@@ -354,6 +356,10 @@ public void afterMove(){
             initCount++;
             floor.setText("B"+initCount);
             init();
+        }else if(chara.getPosX() == enemy.getPosX() && chara.getPosY() == enemy.getPosY()){
+            se.play();
+            Wiz.setImage(new Image (getClass().getResourceAsStream("pic/cat.jpg")));
+            timeline.stop();
```
Comment (equal-l2) [inline MapGameController.java:362]: Why does it stop?

**Review by equal-l2 — CHANGES_REQUESTED** [review]

**Comment by equal-l2** [comment]
@GGBBB2000 Please change the title to more descriptive one.
`Event test` have very little information. 
(Which or what event does it handles? What kind of test does it do?)

File: MapGameController.java (hunk 2 above)
Comment (GGBBB2000) [reply MapGameController.java:362]: I thought  to make it alternative just like invincible time.

Comment (equal-l2) [reply MapGameController.java:362]: You mean that the enemy stops for a while once it touches the player?

Comment (equal-l2) [reply MapGameController.java:362]: In the current implementation, the enemy keep stopped until the player proceeds to the next floor.

Comment (GGBBB2000) [reply MapGameController.java:362]: Yes. 
But, I didn' t have idea after stopping, so I  did'nt anything.

Comment (equal-l2) [reply MapGameController.java]: Probably you want to make new boolean field to keep the state.

**Comment by Aring8510** [comment]
大した問題じゃなさそうなのでマージしてもいいですか

**Comment by Aring8510** [comment]
equalくんには悪いけど僕の権限をもってマージしちゃいました。

**Comment by equal-l2** [comment]
最終決定権はOwnerにあるので全然OKです

**Pull request merged**